Data/Sys/GameSettings/GVSJ8P.ini

```diff
@@ -1,5 +1,4 @@
-# GVS46E, GVS46J - Virtua Striker 4 ver. 2006
-# Because Triforce games have weird IDs, properties are inherited from GVS.ini (Virtua Striker 3 ver. 2002)!
+# GVSJ8P, Virtua Striker 2002 (GC), Virtua Striker 2002 (Triforce), Virtua Striker 4, Virtua Striker 4 ver. 2006
```

Comment:
You have deleted the GVS.ini but added the GC title here? I suspect you were to rename this file to GVS.ini

The title comment should read like this after you've renamed this file
```suggestion
# GVSP8P, GVSE8P, GVSJ8P - Virtua Striker 2002 (GC), Virtua Striker 2002 (Triforce), Virtua Striker 4, Virtua Striker 4 ver. 2006
```

Reply:
The current GVS.ini is for the triforce version regardless, at least according to the name, which is why I deleted it.

Reply:
A 3 letter GameID will apply to all GameIDs who's first 3 letters match it. So the GVS.ini applies to these Triforce games and all regions of the GameCube Virtua Striker 2002.

GVSJ8P.ini will only apply to these Triforce games and the Japanese release of GameCube title Virtua Striker 2002 called "Virtual Striker 3 Ver. 2002" in that region.

**The problem is FPRF won't be set for either the NTSC or PAL versions of Virtua Striker 2002 if the only GameID you've left is GVSJ8P.ini**

Reply:
Is that intentional? Is it only the Triforce Virtua Striker requiring FPRF?

Reply:
I have no idea, these are really old .ini files.

Reply:
In that case, you should keep GVS.ini.